spec/infinispan_cluster_spec.js

```diff
@@ -5,7 +5,7 @@ var t = require('./utils/testing'); // Testing dependency
 var tests = require('./tests'); // Shared tests
 
 describe('Infinispan cluster client', function() {
-  var client = t.client(t.cluster1);
+  var client = t.client(t.cluster1, t.authOpts);
```

Comment:
Couldn't the authOpts be an optional parameter? I would expect only one spec to actually use authentication, especially PLAIN authentication.

Reply:
no. I need to test authentication in every scenario. there is many code where the topology changes and authentication has impacted the code everywhere. there are tests that is not mandatory, but this implies using another infinispan.xml for those tests, and running a non auth standalone server